Nemani/template/simple_functions.py

```diff
@@ -10,3 +10,17 @@ def factorial(value):
         return 1
     else:
         return value * factorial(value - 1)
+
+def is_prime(n):
+
+    if num > 1:
+
+   for i in range(2,num):
```

Comment:
The indentation here looks incorrect, but it might be my browser.
Could you double check?